### DISCUSSION TITLE
Stuck with predefined variables?

### DISCUSSION THREAD

**Discussion by sebnapower** [haugene/docker-transmission-openvpn#2511] [General]
Hi All,

First of all I am loving the project so thank you for bringing it to life :)

I am using docker image on Synology DSM 6 and now 7 for about a year. With previous version I believe we were not able to change some settings in the internal Transmission Contol Panel so I had to define a variable in Docker - for example for number of concurrent downloads and it was not pretty but effective. 

With DSM 7 and current version of image we can now change and save those settings from within Transmission's CP but before realizing it I have spun the Docker image with pre-defined value for concurrent downloads and with either this new version or DSM 7 adjusting variables in Docker once the machine is installed is broken as no changes can be saved. 

They cannot be saved because by default some configuration options are empty and now DSM and Docker does not allow for saving any empty options and as such fails to save any options. So I am stuck with concurrent downloads options I pre-defined in build manifest and even tough I can change it from within Transmission's CP after restart of container it goes back to what I have pre-defined when creating container in the first place.

Is it a known bug? Is there a fix? Is there a workaround? 

Many thanks my friends. 

EDIT: I am using 4.3.0 image.

**Comment by pkishino** [comment]
delete container, start again.. all transmission settings can be edited and saved

**Reply by sebnapower** [reply]
I will post a screenshot of the problem.
